### PR TITLE
Add style for curly braces around hash params

### PR DESCRIPTION
**Pull request by TheBlackArroVV**
https://github.com/datarockets/datarockets-style/issues/73
https://rubocop.readthedocs.io/en/latest/cops_style/#stylebracesaroundhashparameters

### CODE REVIEW

File: STYLE_GUIDE.md [hunk 1]
```diff
@@ -272,6 +272,20 @@ end
   So set up it on the local rubocop config manually.
   <sup>[[link](#style-magic-link)]</sup>
 
+* <a name="style-braces-around-hash-params"></a>
+  Write empty methods in an expanded way.
```
Comment (roman-dubrovsky) [inline STYLE_GUIDE.md]: Looks like somebody forgot to change the copied-pasted text 😉

Comment (TheBlackArroVV) [reply STYLE_GUIDE.md]: Oops

File: STYLE_GUIDE.md [hunk 2]
```diff
@@ -272,6 +272,20 @@ end
   So set up it on the local rubocop config manually.
   <sup>[[link](#style-magic-link)]</sup>
 
+* <a name="style-braces-around-hash-params"></a>
+  Write hash params in contentxt dependent way.
```
Comment (roman-dubrovsky) [inline STYLE_GUIDE.md]: > contentxt dependent 

what is `contentxt dependent way`? This is ok for change log but not for style guide (it should understand the guy who doesn't know about rubocop)

File: STYLE_GUIDE.md [hunk 3]
```diff
@@ -272,6 +272,20 @@ end
   So set up it on the local rubocop config manually.
   <sup>[[link](#style-magic-link)]</sup>
 
+* <a name="style-braces-around-hash-params"></a>
+  We prefer to write hash parameters in style dependent to interfaces public API.
```
Comment (roman-dubrovsky) [inline STYLE_GUIDE.md]: eh, this still isn't clear.

Anyway in the [cop page](https://rubocop.readthedocs.io/en/latest/cops_style/#stylebracesaroundhashparameters), he have a good description for this rule

```
# The `context_dependent` style checks that the last parameter
# doesn't have braces around it, but requires braces if the
# second to last parameter is also a hash literal.
```

So let's re-phrase it and use it in our style guide 

```
Not to use braces around the last hash literal parameter. 
But requires braces if the second to last parameter is also a hash literal
```

Thoughts?

**Comment by roman-dubrovsky** [comment]
+ rebase pls you changes and verify that changelog note added to an unreleased block

File: STYLE_GUIDE.md (hunk 3 above)
Comment (TheBlackArroVV) [reply STYLE_GUIDE.md]: Looks good to me

**Pull request merged**